pages/loginPage.js

```diff
@@ -0,0 +1,7 @@
+var loginPage = function() {
```

Comment:
The same here; loginPage file should have loginPage class which extends the class Pages
Inside the class, you will have getters, where you should get all locators used in the methods below.
Under the getters section inside the class loginPage you will have the actions part, which includes all methods releated to the certain page. For example; the method "logIn" is the method that will be used when running 99% of your tests. Most tests require logging in to the page, so a method will be created that will include the flow of logging in to the page, from entering the email, password, clicking on the login button. and validating that the page has been logged in.
Example:
 logIn(email,password,waitForCategories=true){
    // waitForCategories=data.booleanTrue means that this method should wait for Categories to load on lading page, if log in is successsful, and if log in isn't successfull, then method shoulnd't wait for Categories, because they won't show
        console.log("This method sends data do Email, Password and clicks on login button");
        return this.emailField.sendKeys(email)
            .then(() => this.passwordField.sendKeys(password))
            .then(() => this.loginButton.click())
            .then(() => { if(waitForCategories) homePage.waitForCategories()} )
            .then(() => browser.sleep(2000))